ci/component.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.16.1
+    tag: 1.16.6
```

Comment:
shouldn't this be just 1.16 and then the latest sub version is pulled in ?

Reply:
When looking at all the other repos, the patch number is also given. For example; https://github.com/ONSdigital/dp-topic-api/blob/develop/ci/build.yml#L9